petric.py

```diff
@@ -134,6 +135,12 @@ def evaluate(self, test_im: STIR.ImageData) -> dict[str, float]:
             for voi_name, voi_indices in sorted(self.voi_indices.items())}
         return {**whole, **local}
 
+    def keys(self):
+        l = ["RMSE_whole_object", "RMSE_background"]
+        for voi_name in sorted(self.voi_indices.keys()):
+            l.append(f"AEM_VOI_{voi_name}")
+        return l
+
```

Comment:
seems unused, but if you need it...

```suggestion
    def keys(self):
        return ["RMSE_whole_object", "RMSE_background"] + [f"AEM_VOI_{voi_name}" for voi_name in sorted(self.voi_indices)]

```